package.json

```diff
@@ -61,9 +58,8 @@
     "open": "0.0.5",
     "promzard": "0.3.0",
     "read": "1.0.7",
-    "request": "2.67.0",
```

Comment:
No. I added that plugin and configured it, and then added the **request** dependency back and ESLint still passed, so it doesn't seem to do the same thing as depcheck.

We can probably just use **depcheck** either on Travis-CI only, or add it to devDependencies.